lerobot/common/robot_devices/motors/dynamixel.py

```diff
@@ -359,6 +362,11 @@ def connect(self):
                 f"DynamixelMotorsBus({self.port}) is already connected. Do not call `motors_bus.connect()` twice."
             )
 
+        if self.mock:
```

Comment:
Nit: It seems like what is being "mocked" is the `dynamixel_sdk`... not the `dynamixel` class itself. Should this class be called `mock_dynamixel_sdk`?

Reply:
Indeed!
- Renamed `mock_dynamixel.py` to `mock_dynamixel_sdk.py`
- Renamed `mock_intelrealsense.py` to `mock_pyrealsense2.py`
- Renamed `mock_opencv.py` to `mock_cv2.py`

Reply:
Awesome!